tensorboard/plugins/pr_curve/summary.py

```diff
@@ -22,13 +22,17 @@
 from __future__ import division
 from __future__ import print_function
 
+import numpy as np
 import tensorflow as tf
 
 from tensorboard.plugins.pr_curve import metadata
 
 # A value that we use as the minimum value during division of counts to prevent
-# division by 0. 1 suffices because counts of course must be whole numbers.
-_MINIMUM_COUNT = 1.0
+# division by 0.
+_MINIMUM_COUNT = 1e-7
```

Comment:
I don't understand the reason for this change. Why does `1` no longer suffice?

Reply:
Ah, I added a comment.